### ISSUE TITLE
Add Go Around Function

### ISSUE THREAD

**Issue by CaptCrystal** [openscope/openscope#2139]
Browser: Microsoft Edge

Airport: EGGC

Approximate play time before issue: N/A (no issue)

#### Issue Description
No issue

#### Steps to reproduce

No issue
-

**Comment by airman1994** [comment]
Absolutely! Lack of "Go-around" and "Execute Missed Approach" commands is devastating.
